src/patternfly/components/SimpleList/simple-list.scss

```
@@ -57,13 +67,17 @@
   &:is(:hover, :focus) {
     --#{$simple-list}__item-link--BackgroundColor: var(--#{$simple-list}__item-link--hover--BackgroundColor);
     --#{$simple-list}__item-link--Color: var(--#{$simple-list}__item-link--hover--Color);
+    --#{$simple-list}__item-link--BorderWidth: var(--pf-t--global--border--width--regular);
+    --#{$simple-list}__item-link--BorderColor: var(--pf-t--global--border--color--high-contrast);
```

As a general rule, we only use component vars (not tokens) in the style declarations for a component. We do all of the token assigning at the top of the stylesheet. So you could add this to the vars at the top of the stylesheet:

```
    --#{$simple-list}__item-link--hover--BorderWidth: var(--pf-t--global--border--width--regular);
```

Then you can update this block to:

```suggestion
    --#{$simple-list}__item-link--BorderWidth: var(--#{$simple-list}__item-link--hover--BorderWidth);
```